json-schema/complianceSmsDevice.json

```diff
@@ -0,0 +1,41 @@
+{
+  "description": "Holds information about an SMS \"device\" (an individual SMS channel).",
+  "properties": {
+    "channel": {
```

Comment:
Its weird that some files are snake case and others are camel case

Reply:
yeah I think I'd prefer to fix this in the source. will discuss with the team and copy the changes here before merging